src/pages/PrepareCondition/index.tsx

```diff
@@ -737,8 +758,14 @@ export const PrepareCondition = () => {
               outcome={outcome}
               outcomes={outcomes}
               removeOutcome={removeOutcome}
+              toggleEditOutcome={toggleEditOutcome}
               updateOutcome={updateOutcome}
             />
+            {areOutcomesBeingEdited && !submitDisabled && (
+              <ErrorContainer>
+                <ErrorMessage>Unsaved changes</ErrorMessage>
+              </ErrorContainer>
+            )}
```

Comment:
This and everything inside `PrepareCondition`  could be moved inside if we only show a message independent of the submit button state.